pytest_fixtures/broker.py

```diff
@@ -67,10 +67,19 @@ def rhel7_contenthost_module():
 @pytest.fixture
 def rhel8_contenthost():
     """A fixture that provides a content host object based on the rhel8 nick"""
+    pytest.skip('Skipped until rhel8 templates have wget added')
     with VMBroker(nick='rhel8', host_classes={'host': ContentHost}) as host:
         yield host
 
 
+@pytest.fixture
+def rhel6_contenthost():
+    """A function-level fixture that provides a content host object based on the rhel6 nick"""
+    pytest.skip('Waiting on RHEL 6 SatLab templates')
+    with VMBroker(nick='rhel6', host_classes={'host': ContentHost}) as host:
```

Comment:
Use `DISTRO_RHEL6`, `DISTRO_RHEL7`, and `DISTRO_RHEL8` from `robottelo.constants`.